src/displayapp/screens/ApplicationList.cpp

```diff
@@ -33,47 +37,41 @@ bool ApplicationList::Refresh() {
   return running;
 }
 
-bool ApplicationList::OnButtonPushed() {
-  running = false;
-  app->StartApp(Apps::Clock);
-  return true;
-}
-
 bool ApplicationList::OnTouchEvent(Pinetime::Applications::TouchEvents event) {
   return screens.OnTouchEvent(event);
 }
 
 std::unique_ptr<Screen> ApplicationList::CreateScreen1() {
   std::array<Screens::Tile::Applications, 6> applications {
-          {{Symbols::clock, Apps::Clock},
-          {Symbols::music, Apps::Music},
-          {Symbols::sun, Apps::Brightness},
-          {Symbols::list, Apps::SysInfo},
-          {Symbols::check, Apps::FirmwareValidation},
-          {Symbols::heartBeat, Apps::HeartRate}
+          {
+            {Symbols::stopWatch,    Apps::StopWatch},
+            {Symbols::music,        Apps::Music},
+            {Symbols::map,          Apps::Navigation},
+            {Symbols::shoe,         Apps::Clock},
```

Comment:
Is it intentional that the "shoe application" returns to Clock?

Reply:
Maybe that's a place holder for the new app I added in the motion PR that displays the value of the axis from the accelerometer ?

Reply:
yes, is for the app to show the axis, or in the future to show the steps...